app/rnames_app/templates/relation_sql_detail.html

```diff
@@ -9,8 +9,8 @@
           <a class="btn btn-default" href="{% url 'relation-edit' pk=relation.pk %}"><span class="fa fa-pencil-square-o"></span></a>
           <a class="btn btn-default" href="{% url 'relation-delete' pk=relation.pk %}"><span class="fa  fa-trash-o"></span></a>
         {% endif %}
-        <p><a href="{{request.META.HTTP_REFERER}}">Go back</a></p>
```

Comment:
This row should likely be unchanged since this change was introduced in 47f6cd6c36818a6562831d4ed3076bcbfefd2fed and reverted in d50c41a279bb485e14f5c198c2b8b0f1f7c88144